uint16 id
sweetiebot_cv_lite/Rect rect
